apps/condo/domains/marketplace/schema/fields/price.js

```diff
@@ -0,0 +1,82 @@
+const Ajv = require('ajv')
+
+const { render, getGQLErrorValidator } = require('@condo/domains/common/schema/json.utils')
+const { ERROR_INVALID_PRICE, VAT_OPTIONS } = require('@condo/domains/marketplace/constants')
+const PRICE_GQL_TYPE_NAME = 'PriceSchemaField'
+const PRICE_GQL_INPUT_NAME = 'PriceSchemaFieldInput'
```

Comment:
These names are global for the whole schema. I suggest to use a template to name such types:
`<ModelName><FieldName>SchemaField[Input]`
```suggestion
const PRICE_GQL_TYPE_NAME = 'MarketItemPricePriceSchemaField'
const PRICE_GQL_INPUT_NAME = 'MarketItemPricePriceSchemaFieldInput'
```